README.md

```diff
@@ -1,3 +1,18 @@
 # lyant - Learn You A Next Thing
 
 Bored? Want to learn something?
+
+Added github login, to run the project just open `app->controller->app.go`
```

Comment:
I would prefer we don't store the defaults/config in code like this. Instead we should load the secrets from environment variables in the revel configuration file.   There could/should be a setup script or readme documentation about what environment variables are required.